docs/advanced-usage.md

```diff
@@ -249,3 +268,31 @@ steps:
 ```
 
 NOTE: As per https://github.com/actions/setup-node/issues/49 you cannot use `secrets.GITHUB_TOKEN` to access private GitHub Packages within the same organisation but in a different repository.
+
+## Automatically enable `corepack`
+
+_Requires Node >=14.x_
+
+`corepack` is a new experimental feature that automatically installs package managers as they are used. [Read more about it here](https://nodejs.org/docs/latest/api/corepack.html)
+
+```yaml
+steps:
+- uses: actions/checkout@v3
+- uses: actions/setup-node@v3
+  with:
+    node-version: '14.x'
+    corepack: enable
```

Comment:
Should this be “true”. Or perhaps it should allow arbitrary args to be passed to corepack CLI?

Reply:
I'm so used to writing `corepack enable` in the CLI that I wrote it here too 😅